source/catalog.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Каталог</title>
+</head>
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo" href="#">
```

Comment:
```suggestion
    <a class="main-header__logo" href="index.html">
```
Так как это ссылка на главную страницу.